packages/nvidia_nat_maf/pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools >= 64", "setuptools-scm>=8"]
+
+
+[tool.setuptools.packages.find]
+where = ["src"]
+include = ["nat.*"]
+
+
+[tool.setuptools_scm]
+git_describe_command = "git describe --long --first-parent"
+root = "../.."
+
+
+[project]
+name = "nvidia-nat-maf"
+dynamic = ["version"]
+dependencies = [
+  # Keep package version constraints as open as possible to avoid conflicts with other packages. Always define a minimum
+  # version when adding a new package. If unsure, default to using `~=` instead of `==`. Does not apply to nvidia-nat packages.
+  # Keep sorted!!!
+]
```

Comment:
```suggestion
dependencies = [
  # Keep package version constraints as open as possible to avoid conflicts with other packages. Always define a minimum
  # version when adding a new package. If unsure, default to using `~=` instead of `==`. Does not apply to nvidia-nat packages.
  # Keep sorted!!!
  # 
  # MAF is currently in beta, replace with a release version once we have one
  "agent-framework~=1.0.0b"
]
```